### DISCUSSION TITLE
Meeting: May 7, 2024 agenda

### DISCUSSION THREAD

**Discussion by earth2marsh** [OAI/sig-moonwalk#109] [Announcements]
## Weekly meetings happen on Tuesdays

**Time:** 5pm GMT / 9am Pacific
**Zoom:** [https://zoom.us/j/975841675](https://zoom.us/j/975841675?pwd=SUh4MjRLaEFKNlI3RElpWTdhRDVVUT09), dial-in passcode: 763054

### Accessibility & Etiquette

* Participants must abide by our [Code-of-Conduct](https://github.com/OAI/OpenAPI-Specification/blob/main/CODE_OF_CONDUCT.md#code-of-conduct).
* Meetings are recorded for future reference, and for those who are not able to attend in-person.
* We invited challenges to language or behaviour that is harmful or not inclusive during this meeting.
* We look forward to your participation, but please consider these acts of etiquette:
  * Remain on mute when not speaking to prevent interruptions.
  * Blur your background to reduce visual distractions.
  * Use the Zoom meeting "Raise Hand" feature to notify the group when you wish to speak.

| Blur My Background | Raise Hand |
|-|-|
| <img width="323" alt="Screenshot of Zoom UI showing the 'Stop Video' and 'Blur My Background' control" src="https://github.com/OAI/OpenAPI-Specification/assets/7367/7e43dbbb-6529-46e6-8b04-4c1aa852d9dd"> | <img width="323" alt="Screenshot of Zoom UI showing the 'Reaction' and 'Raise Hand' control" src="https://github.com/OAI/OpenAPI-Specification/assets/7367/f991722f-4651-40aa-9bc4-7e9a2a165a6a"> |

### Agenda Structure

| Topic | Owner | Decision/NextStep |
|-------|---------|---------|  
| | | |
Intros and governance meta-topics (5 mins) | SIG members | -- |
General discussion topics (list below) | SIG members | -- |

Whether attending or not, **anyone can comment on this issue prior to the meeting to suggest topics or to add comments on planned topics or proposals**.

**Comment by mikekistler** [comment]
I'd like to revisit the proposal for parameter interdependencies in this meeting.

https://github.com/OAI/sig-moonwalk/discussions/100

**Comment by ralfhandl** [comment]
I'd like to continue the "signature" discussion, which as a sub-topic includes the parameter interdependencies.

**Reply by earth2marsh** [reply]
Gary added some comments to this doc we started last week: https://docs.google.com/document/d/1OM9MuiuafpiX2R-cNIhvaf7xJH95sXH7JtsMS0erxfQ
